runtime/gc_base/StringTable.cpp

```diff
@@ -556,9 +594,19 @@ j9gc_createJavaLangString(J9VMThread *vmThread, U_8 *data, UDATA length, UDATA s
 					break;
 				}
 			}
+			isASCIIorLatin1 = isASCII;
+			if (!isASCII && compressStrings && J2SE_VERSION(vm) >= J2SE_V17) {
```

Comment:
The test of `compressStrings` is redundant here (see line 589).

Reply:
Fixed.